documentation/manual/src/main/asciidoc/modules/infinispan.asciidoc

```diff
@@ -210,6 +210,11 @@ The following two strategies exist (values of the `org.hibernate.ogm.datastore.k
 
 +
 Defaults to `CACHE_PER_TABLE`. It is the recommended strategy as it makes it easier to target a specific cache for a given entity.
+`hibernate.ogm.datastore.create_database`::
+Accept `true` or `false`. If `true`, Hibernate OGM will create a new cache on an Infinispan Server
+when the required one is not defined, this works only if a default configuration is defined on
+the server. If set to `false` an exception is thrown when the cache is not defined.
+Defaults to `false`.
```

Comment:
May I suggest:

`hibernate.ogm.datastore.create_database`::
If set to `true` Hibernate OGM will create any missing Cache definitions on the Infinispan Server.
This requires the Infinispan Server configuration to have a default configuration defined, as this will be copied to the newly defined caches.
If set to `false` an exception is thrown when a Cache is expected but was not explicitly configured on the server.
Defaults to `false`.